RallyHereIntegration/Source/RallyHereIntegration/Private/RH_SettingsSubsystem.cpp

```diff
@@ -0,0 +1,24 @@
+#include "RH_SettingsSubsystem.h"
+
+URH_SettingsSubsystem::URH_SettingsSubsystem(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+}
+
+void URH_SettingsSubsystem::GetSettingTypes(const FRH_GenericSuccessBlock& Delegate)
+{
+	UE_LOG(LogRallyHereIntegration, VeryVerbose, TEXT("[%s]"), ANSI_TO_TCHAR(__FUNCTION__));
+
+	auto Request = TGetConfigSettings::Request();
+	Request.AuthContext = GetAuthContext();
+
+	if (!TGetConfigSettings::DoCall(RH_APIs::GetSettingsAPI(), Request, TGetConfigSettings::Delegate::CreateLambda([this, Delegate](const TGetConfigSettings::Response& Resp)
+		{
+			CachedSettingsTypes.Empty();
```

Comment:
This needs to do some validation checking (IsSuccessful, etc).

Would suggest looking into the FRH_SimpleQueryHelper class (config subsystem is a good example).  It takes two lambas, one for the update logic (only triggered on success), and one for completion (with error information, and uses FRH_GenericSuccessBlockWithError).  Would suggest converting this function itself to use the FRH_GenericSuccessBlockWithError, as I have been slowly converting things to use it as it provides more information about why something failed.